CONTRIBUTING.md

```diff
@@ -208,6 +208,48 @@ This prevents nesting levels from getting deeper then they need to be.
 
 Math operators like +, -, /, *, etc are up in the air, just choose which version looks more readable.
```

Comment:
this is mentioned in the "Other Notes" section already, and isn't really a DM-specific quirk.
(also multiplication over division is only preferred when the operation terminates in decimal space (the resulting number is not recurring))